k9mail/src/main/java/com/fsck/k9/activity/compose/RecipientMvpView.java

```diff
@@ -202,7 +202,13 @@ public void setCcVisibility(boolean visible) {
     }
 
     public void setBccVisibility(boolean visible) {
+        boolean bccAlreadyVisible = isBccVisible();
         bccWrapper.setVisibility(visible ? View.VISIBLE : View.GONE);
+        if(visible && !bccAlreadyVisible){
```

Comment:
I agree, this a workaround not a fix.
It was caused by the tokenautocomplete external library which bugs out when the editbox containing  data is made visible.

I can put this whole removing and re-adding thing in the recipientPresenter class
or shall i work on a fix?